packages/sign-client/src/client.ts

```diff
@@ -49,6 +62,23 @@ export class SignClient extends ISignClient {
     this.proposal = new Proposal(this.core, this.logger);
     this.pendingRequest = new PendingRequest(this.core, this.logger);
     this.engine = new Engine(this);
+    this.auth = {
+      authKeys: new Store(
```

Comment:
Nit: would be nicer to abstract these like with `Session`, `Proposal` stores etc but not pressing if this is the only occurrence of these stores atm